src/main/java/org/fcrepo/camel/FcrepoConfiguration.java

```diff
@@ -0,0 +1,278 @@
+/**
+ * Copyright 2014 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.camel;
+
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriParams;
+import org.apache.camel.RuntimeCamelException;
+
+/**
+ * An FcrepoConfiguration class.
+ * @author Aaron Coburn
+ * @since Jan 20, 2015
+ */
+@UriParams
+public class FcrepoConfiguration implements Cloneable {
+
+    private String baseUrl = "";
+
+    @UriParam
+    private String contentType = null;
+
+    @UriParam
+    private String accept = null;
+
+    @UriParam
+    private String transform = null;
+
+    @UriParam
+    private String authUsername = null;
+
+    @UriParam
+    private String authPassword = null;
+
+    @UriParam
+    private String authHost = null;
+
+    @UriParam
+    private Boolean tombstone = false;
+
+    @UriParam
+    private Boolean metadata = true;
+
+    @UriParam
+    private Boolean throwExceptionOnFailure = true;
+
+    @UriParam
+    private String preferInclude = null;
+
+    @UriParam
+    private String preferOmit = null;
+
+    @UriParam
+    private Boolean secure = false;
+
+    /**
+     * Create a new FcrepoConfiguration object
+     */
+    public FcrepoConfiguration() {
+        super();
+    }
+
+    /**
+     * Copy an FcrepoConfiguration object.
+     */
+    @Override
+    public FcrepoConfiguration clone() {
+        try {
+            return (FcrepoConfiguration) super.clone();
```

Comment:
I was following the pattern in some existing camel components. See, for example, the [SSH Component](https://github.com/apache/camel/blob/master/components/camel-ssh/src/main/java/org/apache/camel/component/ssh/SshConfiguration.java#L82) and the [JMS Component](https://github.com/apache/camel/blob/master/components/camel-jms/src/main/java/org/apache/camel/component/jms/JmsConfiguration.java#L216)

In those examples, the function is called `copy()`, but I used `clone()` here, following the documentation of [Object.clone](http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#clone())

Is there a better way to do this?